salt/modules/junos.py

```diff
@@ -2050,6 +2050,7 @@ def _make_source_list(dir):
     return dir_list
 
 
+# pragma: no cover
```

Comment:
Skipping functionality related to Junos native salt-minion, we not going to be using it, so no coverage, since don't have the native salt-minion to test it with

Reply:
But doesn't this comment disable comment on the whole file(since it's on it's own line)?

Reply:
I believe this relates to the function, since just before it